src/components/GenericErrorView/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '../Icon';
+import defaultTheme from '../../styles/themes/default';
+import * as Expensicons from '../Icon/Expensicons';
+import ExpensifyText from '../ExpensifyText';
+import Button from '../Button';
+import LogoWordmark from '../../../assets/images/expensify-wordmark.svg';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+import compose from '../../libs/compose';
+import variables from '../../styles/variables';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import ErrorBodyText from './ErrorBodyText';
+import TextLink from '../TextLink';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
+};
+
+const GenericErrorView = (props) => {
+    const refreshPage = () => {
+        Navigation.navigate(Navigation.getCurrentRoute());
+    };
```

Comment:
Better move this to navigation lib itself if this works for all platforms. `Navigation.refresh`.

Reply:
Were you able to refresh the page on all platforms?

Reply:
Not yet. Will get back to it today.

Reply:
@parasharrajat PR updated